src/ServiceControl/DbMigrations/1.41.3/RerunClassifiersMigration.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Particular.ServiceControl.DbMigrations
+{
+    using global::ServiceControl.Recoverability;
+    using Raven.Client;
+
+    public class RerunClassifiersMigration_1_41_3 : IMigration
+    {
+        public string Apply(IDocumentStore store)
+        {
+            using (var session = store.OpenSession())
+            {
+                var reclassifySettings = session.Load<ReclassifyErrorSettings>(ReclassifyErrorSettings.IdentifierCase);
+
+                if (reclassifySettings != null)
+                {
+                    reclassifySettings.ReclassificationDone = false;
```

Comment:
This is only a problem if ServiceControl is not updated via the installer, correct? Doesn't the upgrade process run the migrations meaning by the time SC starts up the data will be updated already?

I'll verify first though...

It might still be worth changing the reclassification process to append instead of re-classifying _all_ of the existing classifications.